jobbergate-cli/jobbergate_cli/constants.py

```diff
@@ -0,0 +1,35 @@
+"""
+Constants used throughout the tool
+"""
+from pathlib import Path
+
+
+JOBBERGATE_APPLICATION_CONFIG = {
+    "application_name": "",
+    "application_description": "",
+    "application_file": "",
+    "application_config": "",
+}
+
+JOBBERGATE_JOB_SCRIPT_CONFIG = {
+    "job_script_name": "",
+    "job_script_description": "TEST_DESC",
+    "job_script_owner": "",
+    "application": "",
+}
+
+JOBBERGATE_JOB_SUBMISSION_CONFIG = {
+    "job_submission_name": "",
+    "job_submission_description": "TEST_DESC",
+    "job_submission_owner": "",
+    "job_script": "",
+}
+
+JOBBERGATE_DEFAULT_DOTENV_PATH = Path("/etc/default/jobbergate-cli")
+JOBBERGATE_APPLICATION_MODULE_FILE_NAME = "jobbergate.py"
+JOBBERGATE_APPLICATION_CONFIG_FILE_NAME = "jobbergate.yaml"
+TAR_NAME = "jobbergate.tar.gz"
```

Comment:
I think each file should be named diferrently, otherwise, the files can be overwritten. What about something like: `<user-id>-<datetime>.tar.gz`?

Reply:
the user_id in this case can be the same one used for Sentry. Thoughts?

Reply:
The files don't collide because the application id is included in the path.

See the `key_template` attribute in jobbergateapi2/s3_manager.py.

Reply:
even so, different users can face different problems with the same application. Identifying the user is useful I guess. Thoughts?

Reply:
The ownership of the application is bound in the database. We shouldn't include it in the path on S3. We were doing this before, and it caused problems because if one user updated an application owned by a different user, it would create a new entry with a different user id but the same application id. Since the application id is unique to an instance of the application, we should include only that in the application's s3 path. If there are issues with an application (indicated by an error log), we should consult the database if we need to learn who the owner is. Beyond that, the logs are collected user-by-user so there will be breadcrumbs there to indicate who had the issue.

Reply:
I agree, there is no need to make a unique filename here.

Reply:
I see your point @matheushent , but it adds unneeded complexity. The value of creating a named file here will only serve developers when they look in s3, it will not add value to the code as far as I can see. Resolving this issue so we can move forward.